.snippets/text/builders/ethereum/precompiles/ux/batch/batch-parameters.md

```diff
@@ -0,0 +1,4 @@
+- ***address[]* to** - an array of addresses to direct subtransactions to, where each entry is a subtransaction
```

Comment:
I think what happened here was that we used to use this in two places, but it looks like now only the Batch tutorial  uses it. Can you please delete the snippet and just move this stuff to that file instead?